### DISCUSSION TITLE
Badge with Prefix

### DISCUSSION THREAD

**Discussion by josh-i386g** [mantinedev/mantine#6066] [Feature requests]
I am unsure if you guys fancy this but while using badge i thought of it having prefixes.

example use case is for items that have multiple badges where two distinct keys have similar values.

example where left is default and right is filled-black

![image](https://github.com/mantinedev/mantine/assets/150557754/82a01e92-1373-40d6-b24c-65d92dbdf413)

example from https://shields.io/

![](https://img.shields.io/badge/something-like_this-green)

another one i found is https://badgen.net/

![](https://badgen.net/static/license/MIT/blue)

Just an idea, thank you very much for the elegant components you guys rock.

**Comment by rtivital** [comment]
https://mantine.dev/core/badge/#left-and-right-sections
